<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stick Arena</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/assets/icon.png</normaloff>:/assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="user_list">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>210</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="client_out">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>521</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="openLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="client_in">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>641</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="game_list">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>10</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="joinGame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>455</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>JOIN GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createGame">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CREATE GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logOut">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>570</y>
      <width>174</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>LOG OUT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LobbyBtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>530</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK TO LOBBY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="kickCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Kick</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkWalk">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Walk</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkSuicide">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>570</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Suicide</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="altids">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>570</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
